static/docs/tutorials/deep/preparation.md

```diff
@@ -49,17 +49,6 @@ $ echo ".env/" >> .gitignore
 $ pip install -r code/requirements.txt
 ```
 
-## Install DVC
-
-You'll need DVC on your system to continue this tutorial. Please refer to
-[Installation](/doc/install) for the easiest way to install DVC on your system.
-For this tutorial, installing DVC with [pip](https://pip.pypa.io/) is probably
-the easiest:
-
-```dvc
-$ pip install dvc
-```
-
```

Comment:
Actually this part I agree should be removed BTW, in fact I'm also doing so in #763.